.github/workflows/erlang.yml

```diff
@@ -10,8 +10,8 @@ jobs:
 
     strategy:
       matrix:
-        otp: ['22.3', '23.3', '24.0']
-        rebar: ['3.16.1']
+        otp: ['22.3', '23.3', '24.3']
+        rebar: ['3.18.0']
         include:
         - otp: '21.3'
           rebar: '3.15.2'
```

Comment:
We should remove this one, @pablocostass … 3 versions for tests are more than enough.

Reply:
I wasn't sure whether hank was one of those projects that support a lot of versions or not. On it!